requirements.txt

```diff
@@ -0,0 +1,37 @@
+astroid==2.0.4
```

Comment:
Nice work flow, Not sure if u used all the modules in the requirements.txt file, you can remove the ones your app doesn't use.